### PR TITLE
Python Examples

### PR DESCRIPTION
**Pull request by pavankumarbn**
@lorensen @ajpmaclean Please review these python examples.

### CODE REVIEW

File: src/Python/Graphs/ColorVertexLabels.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import vtk
+def main():
+
```
Comment (ajpmaclean) [inline src/Python/Graphs/ColorVertexLabels.py]: Change
```python
#!/usr/bin/env python
import vtk
def main():
```
to
```python
#!/usr/bin/env python

import vtk


def main():
```

File: src/Python/Graphs/ColorVerticesLookupTable.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import vtk
```
Comment (ajpmaclean) [inline src/Python/Graphs/ColorVerticesLookupTable.py]: Insert a space here

File: src/Python/Graphs/ConstructGraph.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import vtk
```
Comment (ajpmaclean) [inline src/Python/Graphs/ConstructGraph.py]: Insert a space here

File: src/Python/Graphs/ConstructTree.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import vtk
+def main():
```
Comment (ajpmaclean) [inline src/Python/Graphs/ConstructTree.py]: Change
```python
#!/usr/bin/env python
import vtk
def main():
```
to:
```python
#!/usr/bin/env python

import vtk


def main():
``

**Comment by ajpmaclean** [comment]
Please see comments regarding formatting. Make sure Python code follows [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines)

**Comment by pavankumarbn** [comment]
@ajpmaclean Sure I will follow. I have made changes Please let me know if any further changes required.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**